modules/System Admin/i18n_manage_installProcess.php

```diff
@@ -33,14 +33,15 @@
     exit;
 } else {
     //Proceed!
-    if (empty($gibboni18nID)) {
+    if ($gibboni18nID == '') {
```

Comment:
Anytime we're checking POST or GET data it's handy to use an empty check rather than a string check. It handles the couple extra edge cases of zeroes, nulls, or empty arrays being passed in, each of which is invalid in this case.

```suggestion
    if (empty($gibboni18nID)) {
```

Reply:
Ok. I think we need a function to validate the POST and GET parameters in v21, so that we can apply a pattern. What do you think?